### PR TITLE
change references from dev-api to sandbox-api

### PR DESCRIPTION
**Pull request by charleystran**
This PR changes the references od dev-api to sandbox-api for the sandbox migration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,4 +54,4 @@ Adding a new API to the apply page requires changes in a few different places. Y
...
\ No newline at end of file
+Sometimes you will need to test the apply page locally. To do so you can fire up the backend and point the developer portal at it. Clone the `developer-portal-lambda-backend` repo. Run `docker-compose up` and local DynamoDB, Kong and lambda containers will spin up. If you update `.env.local` with `REACT_APP_DEVELOPER_PORTAL_SELF_SERVICE_URL=http://localhost:9000` (You'll need to restart your locally running developer portal) you can test the apply page locally. See the [backend repo](https://github.com/department-of-veterans-affairs/developer-portal-lambda-backend#local-interation) for more information about running the backend locally.
```
Comment (edmangimelli) [inline README.md]: hmmmm. i wonder what's the diff here? throwing these 2 into files, I get nothing when i `diff`

File: src/content/apiDocs/health/argonautDeprecationNotice.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## Deprecation Notice
 
-Consumers currently using the `https://api.va.gov/services/argonaut/v0` (and `https://dev-api.va.gov/services/argonaut/v0`) endpoint, please migrate your applications to use the new Veteran Health API (FHIR) endpoints: `https://api.va.gov/services/fhir/v0/argonaut/data-query/` and `https://dev-api.va.gov/services/fhir/v0/argonaut/data-query/`.
+Consumers currently using the `https://api.va.gov/services/argonaut/v0` (and `https://dev-api.va.gov/services/argonaut/v0`) endpoint, please migrate your applications to use the new Veteran Health API (FHIR) endpoints: `https://api.va.gov/services/fhir/v0/argonaut/data-query/` and `https://sandbox-api.va.gov/services/fhir/v0/argonaut/data-query/`.
 
-Both the legacy API endpoints and this legacy documentation page will no longer be accessible beginning Oct 1, 2019.
\ No newline at end of file
+Both the legacy API endpoints and this legacy documentation page will no longer be accessible beginning Oct 1, 2019.
```
Comment (edmangimelli) [inline src/content/apiDocs/health/argonautDeprecationNotice.mdx]: again. weird. must be trailing whitespace

File: src/mockSwagger.js [hunk 3]
```diff
@@ -7,14 +7,14 @@ export const mockSwagger = {
...
       termsOfService: '',
       contact: { name: 'Vets.gov' },
     },
     tags: [{ name: 'document_uploads', description: 'VA Benefits document upload functionality' }],
     servers: [
       {
-        url: 'dev-api.vets.gov/services/vba_benefits/{version}',
+        url: 'sandbox-apii.vets.gov/services/vba_benefits/{version}',
```
Comment (edmangimelli) [inline src/mockSwagger.js]: typo

File: src/mockSwagger.js [hunk 4]
```diff
@@ -343,14 +343,14 @@ export const mockSwagger = {
...
...als Status\n\n## Background\n\nThis API is being provided as a proof of concept of a general-purpose VA \nAPI gateway. The use case is to allow authorized third-party developers to\nrequest the status of a Veteran's benefit appeals.\n\nThe Appeals API passes data through to Caseflow, a case management system. Caseflow\naccepts a header with the Veteran's SSN and returns the current status of\ntheir appeals.\n\nBecause this application is designed to allow third-parties to request\ninformation on behalf of a Veteran, we are not using VAAFI headers or SSOe.\n\n## Design\n\n### Authorization\n\nAPI requests are authorized by means of a symmetric API token, provided in an HTTP header\nwith name \"apikey\".\n\n### Status Request\n\nAllows a client to check benefit appeals status\n\n1. Client Request: GET https://api.vets.gov/services/appeals/v0/appeals\n    * Provide the Veteran's SSN as the X-VA-SSN header\n    * Provide the VA username of the person requesting the appeals status as the X-VA-User header\n\n2. Service Response: A JSON API object with the current status of appeals\n      \n  ## Reference\n\n  Raw Open API Spec: http://sandbox-api.vets.gov/services/appeals/docs/v0/api\n",
```
Comment (edmangimelli) [inline src/mockSwagger.js]: can this be multiline instead of with `\n` (can we use a template literal instead)?

Comment (edmangimelli) [inline src/mockSwagger.js]: can this be multiline instead of with `\n` (can we use a template literal instead)?

Comment (charleystran) [reply src/mockSwagger.js]: I am sure we could, but maybe a tech debt ticket?

Comment (edmangimelli) [reply src/mockSwagger.js]: ¯\_(ツ)_/¯
if you think that merits a tech debt ticket.
either way is good to me.

**Pull request merged**